### ISSUE TITLE
Serveral md-380 related Problems

### ISSUE THREAD

**Issue by Yrrsinn** [DaleFarnsworth/codeplug#18]
After some DMR absence, I installed the recent binary release (0.15.7) to work on my md-380 codeplug (created with an older EditCp release).

Observed behavior with a TYT MD-380:
- When starting a new codeplug using File -> New... selecting MD-380, 400-480 MHz, a new codeplug is started but under Basic Information the model is a DR780. When trying 
- When downloading a codeplug from a md-380, EditCp reports several invalid fields and the codeplug is for the model DR780 according to Basic Information.
- Older Backups (codeplugs created, edited with older releases of EditCp) show the same errors. The indicated model changed to DR780  as well.

On a first try to understand the issue and solve it by myself, I found this https://github.com/travisgoodspeed/md380tools/issues/41 that is maybe related.

**Comment by DaleFarnsworth** [comment]
Thank you.  All of the behavior you describe I consider as correct behavior.  The model number contained in the codeplug header produced by the Tytera CPS for the MD-380 is DR780.  When you select File->New, I actually load a .rdt file that was produced by the Tytera CPS.  I used to check for the DR780 and display MD-380, but it became complicated as I support many other different models, so I now just display the exact value contained in the codeplug header.  It is very likely that all of the invalid fields reported are actually invalid fields.  When fields contain invalid/illegal values, I flag them and notify the user.  The Tytera CPS silently shows blank values for those fields.  You can examine and fix those fields using editcp or the Tytera CPS.  If you would like, you can send me the codeplug and I'll check the "invalid fields".  IHowever, it would be better for you to try and fix them yourself.

**Comment by DaleFarnsworth** [comment]
Further elaboration on why I show DR780 instead of MD-380.

The MD-380 and the RT3 radios have identical firmware and codeplugs.  They both have DR780 as the model name in the codeplug header.  So when editcp opens one of these codeplugs, there is no way to determine (without asking the user) which radio the codeplug is intended for.  The codeplug will work in either radio model.  I could resolve this by having a version of editcp for the MD-380 and another version for the RT3.  I decided to support multiple radios in a single program.  I also decided that it is not worth asking the user.  However, I do know that the codeplug is usable on either an MD-380 or RT3, so I'm considering displaying the model name as "DR780 (MD-380, RT3)"
